### ISSUE TITLE
Estilo de letras para los comics

### ISSUE THREAD

**Issue by MpaulaL** [Open-Science-Training-Handbook/Open-Science-Training-Handbook_ES#15]
Hola chicos, probablemente a ustedes también les toca traducir algunos de los comics que tienen títulos. Me parece que sería bueno acordar en qué tipo de letra utilizar? - hay alguien que ya haya empezado? Si no, sugiero alguno de los siguientes estilos...por favor revisar la imagen adjunta. Espero sus comentarios antes de proceder con la traducción :-)...gracias y saludos!

![03_educacion vs aprendizaje _es](https://user-images.githubusercontent.com/32437989/45600821-5b0c4880-ba03-11e8-98b5-4374614bc9c1.png)

**Comment by rosariorogel** [comment]
Yo ya envíe todo lo que me tocaba, y en el caso de los textos de los cómics lo que hice fue dejarlas marcadas en formato "quotes" de Github, con la intención de que quien edite las imágenes considere las notas para el nuevo trazado de las imágenes. ¿Cómo están haciendo los demás?

**Comment by rosariorogel** [comment]
No me había dado cuenta que es posible editar las imágenes, me gustó más usar la fuente "Handwrintig Dakota" en mayúsculas, según yo es la que más se parece a la que se usa en las imágenes originales. ¿Qué te parece a ti? ¿y a los demás?

![02_open_policies copia](https://user-images.githubusercontent.com/29648386/45602478-ac590f80-b9e4-11e8-8887-16a59e2fbd10.png)

**Comment by MpaulaL** [comment]
Estimados, mi versión de Windows no tiene Handwriting Dakota :-/...y probablemente los que trabajamos con PC no tendremos acceso, o, alguien tiene algún buen (legal) link para bajar el estilo de letra?

**Comment by cvilchesfurio** [comment]
Concuerdo con Ricardo, va a quedar fantástico! 
Respecto de la letra, Comic Sans parece ser la más común. El beneficio que le veo es que es más gruesa y facilita su visibilidad en las capacitaciones y presentaciones. La Handwriting la encuentro muy delgada y difícil de leer. Que opinan?

**Comment by MpaulaL** [comment]
A mí me facilitaría tremendamente el trabajo si utilizamos Comic Sans MS! Quedo atenta a la opnión del resto :-)

**Comment by rosariorogel** [comment]
Pues como yo ya acabe mi parte, y tengo la fuente "Hand Writing", me ofrezco a:

1. Buscar una versión Bold de la misma fuente
2. Traducir todos los cómics que tienen texto
3. Dejar los cómics ya traducidos para que los responsables de cada apartado los sustituyan

¿Quieren?

**Comment by cvilchesfurio** [comment]
Rosario, sería maravilloso si pudieras traducir los comics a la letra Comic Sans MS. Pienso que es mejor usar un font que tengamos todos (más común), por si más adelante se agregan al Manual otras secciones con nuevas imágenes.

**Comment by rosariorogel** [comment]
Claudia, ya hice el ajuste de fuente (a Cominc Sans) en los cómics del segmento que me corresponde. ¿Quieres que haga el resto o que te apoye en algo más?
Quedo atenta

**Comment by cvilchesfurio** [comment]
Hola Rosario, quisiera confirmar si tus capítulos ya están listos como para comenzar la revisión/edición final... Me cuentas por favor. Muchas gracias y saludos

**Comment by rosariorogel** [comment]
Sí, de hecho si lees mi comentarios en GitHub, te decía que sí, desde hace
más de dos semanas terminé todo lo que me fue asignado.

En otro comentario te expliqué que abrí archivos independientes a los que
les agregué el sufijo "ES", y dejé los "pull request", pero como pasados
unos días no recibí retroalimentación de tu parte, supuse que no había
mayores comentarios, así que yo misma integré mis "pull request" al
"master", ahí los pueden ver todos, ya listos para revisión.

También te pregunté si querías que te apoyara en algo más, me ofrecí a
traducir todos los cómics, pero ya no recibí respuesta, o quizá ayudarle a
alguien. Estoy viendo que algunos tienen problema con el código en las
imágenes y por eso no se despliegan.

Cuenta conmigo por favor para lo que requieras.



*----**Rosario Rogel-Salazar, Dra.*
Universidad Autónoma del Estado de México
Toluca, MEXICO

rosariorogel@gmail.com

Research ID: I-7365-2017

Scopus Author: 35265086900
ORCID: 0000-0002-6018-0635 <https://orcid.org/0000-0002-6018-0635>

Twitter: @rosariorogel <https://twitter.com/rosariorogel>
Skype: rizomaticamente

**Comment by cvilchesfurio** [comment]
Gracias Rosario. He estado con varias cosas que me han impedido atender todos las conversaciones del Foro.
Como puedes ver en el último tema que puse hoy en el Foro, consulto si alguno de los colegas necesita colaboración. Hasta ahora no he tenido novedades. Cualquier cosa te aviso. Muchas gracias y saludos.
